### PR TITLE
Add Shared DocumentDB Cluster

### PR DESCRIPTION
**Pull request by fredericfran-gds**
This documentdb cluster is intended to be used by many
govuk apps that require mongodb.

For now, only asset-manager will use it.

### CODE REVIEW

File: terraform/projects/app-shared-documentdb/main.tf [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+variable "master_password" {
+  type        = "string"
+  description = "Password of master user on DocumentDB cluster"
+}
+
+variable "tls" {
+  type        = "string"
+  description = "Whether to enable or disable TLS for the Licensify DocumentDB cluster. Must be either 'enabled' or 'disabled'."
```
Comment (sengi) [inline terraform/projects/app-shared-documentdb/main.tf]:
```suggestion
  description = "Whether to enable or disable TLS for the DocumentDB cluster. Must be either 'enabled' or 'disabled'."
```

File: terraform/projects/infra-security/main.tf [hunk 2]
```diff
@@ -269,6 +269,11 @@ resource "aws_kms_key" "licensify_documentdb" {
   policy      = "${data.aws_iam_policy_document.kms_sops_policy.json}"
 }
 
+resource "aws_kms_key" "shared_documentdb" {
+  description = "Senstive data in Shared DocumentDB"
```
Comment (sengi) [inline terraform/projects/infra-security/main.tf]:
```suggestion
  description = "Encryption key for Shared DocumentDB"
```

Comment (fredericfran-gds) [reply terraform/projects/infra-security/main.tf]: will fix and above too

File: terraform/projects/app-shared-documentdb/main.tf (hunk 1 above)
Comment (fredericfran-gds) [reply terraform/projects/app-shared-documentdb/main.tf]: Will fix

**Pull request merged**